testing/tests/DevExpress.data/odataStore.tests.js

```diff
@@ -1952,12 +1952,12 @@ QUnit.test("works", function(assert) {
         url: "odata.org",
         responseText: {},
         callback: function(bag) {
-            assert.equal(bag.data.customName, "'customValue'");
+            assert.equal(bag.data.customName, "'customvalue'");
```

Comment:
What if a custom parameter is not related to data but represents a custom value that needs to be transferred verbatim (e.g. base64 encoded data)?

Another concern are string primary keys.
It seems that we must not use `tolower` in `byKey` and for fields that are part of `key`.